assets/js/filter.js

```diff
@@ -0,0 +1,174 @@
+$(function () {
+    // contentApiKey is instance-dependent; replace this with a new key if
+    // testing on a local instance
+    // new SearchinGhostEasy({ contentApiKey: '3ecdb68993794bc0157e776e5d' });
+
+    // Filter feature
+    var primaryFilters = "";
+    var internalFilters = "";
+
+    function getFilterOptions(target) {
+        // Retrieve select value from filter and manipulate the values for filtering
+        var filterOptions = $(target).val() ? $(target).val() : [];
```

Comment:
```suggestion
        var filterOptions = $(target).val() || [];
```

You can use the `||` operator if you're doing something equivalent to `x ? x : y`.

Reply:
wow this is some true js hax